### DISCUSSION TITLE
Welcome to goldfish Discussions!

### DISCUSSION THREAD

**Discussion by jhollway** [stocnet/goldfish#20] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dkarell** [comment]
Hi all,

I'm Daniel Karell, an assistant professor in the Department of Sociology at Yale University. My research is broadly about discursive communities and extremism. I'm currently using goldfish models to examine the growth of online right-wing groups. My primary dataset comprises timestamped YouTube videos and comments; network ties are comments on videos and comments on other comments.

I'm generally curious about how others are approaching multiplex and multilevel networks. (The published papers have been a great start!) I'm also interested in hearing about how others interpret results, especially with complex models. (I often present to audiences accustomed to seeing results as marginal effects, so I think about interpretation in that sense, too.) And, I'm always keen to discuss ideas about socio-semantic networks!

I think it'd be great to have a "users meeting" (or meetings) to share and comment on projects, ask and discuss general questions, and so on. I'm happy to help organize if there is interest.

I'm looking forward to "meeting" you all!

**Comment by alonia** [comment]
这是来自QQ邮箱自动回复邮件。你好，你的邮件已收到。
